app-office/beancount/files/beancount-2.3.3-disable-install-test.patch

```diff
@@ -0,0 +1,31 @@
+diff --git a/beancount/scripts/setup_test.py b/beancount/scripts/setup_test.py
+index 59a165d1..45717643 100644
+--- a/beancount/scripts/setup_test.py
++++ b/beancount/scripts/setup_test.py
+@@ -12,6 +12,10 @@ import sys
+ import tarfile
+ import tempfile
+ import unittest
++import pytest
```

Comment:
But there's no pytest dependency! Are you introducing one just for this? ;)

Reply:
Doh, this is an artifact of originally using the distutil_r1 test thingy